src/analysis/classic/mage/arcane/modules/features/Abilities.tsx

```diff
@@ -7,53 +7,24 @@ class Abilities extends CoreAbilities {
     const combatant = this.selectedCombatant;
     return [
       // Rotational
-      {
-        spell: [SPELLS.ARCANE_BLAST.id, ...SPELLS.ARCANE_BLAST.lowRanks],
-        category: SPELL_CATEGORY.ROTATIONAL,
-        gcd: { base: 1500 },
-        buffSpellId: SPELLS.ARCANE_BLAST_DEBUFF.id,
-      },
       {
         spell: [SPELLS.ARCANE_MISSILES.id, ...SPELLS.ARCANE_MISSILES.lowRanks],
         category: SPELL_CATEGORY.ROTATIONAL,
-        gcd: { base: 1500 },
-      },
-      {
-        spell: [SPELLS.ARCANE_BARRAGE.id, ...SPELLS.ARCANE_BARRAGE.lowRanks],
-        category: SPELL_CATEGORY.ROTATIONAL,
-        gcd: { base: 1500 },
-      },
-      {
-        spell: [SPELLS.ICE_LANCE.id, ...SPELLS.ICE_LANCE.lowRanks],
-        category: SPELL_CATEGORY.ROTATIONAL,
-        gcd: { base: 1500 },
-      },
-      {
-        spell: [SPELLS.FROSTBOLT.id, ...SPELLS.FROSTBOLT.lowRanks],
-        category: SPELL_CATEGORY.ROTATIONAL,
-        gcd: { base: 1500 },
-      },
-      {
-        spell: [SPELLS.FROSTFIRE_BOLT.id, ...SPELLS.FROSTFIRE_BOLT.lowRanks],
-        category: SPELL_CATEGORY.ROTATIONAL,
-        gcd: { base: 1500 },
-      },
-      {
-        spell: [SPELLS.FIREBALL.id, ...SPELLS.FIREBALL.lowRanks],
-        category: SPELL_CATEGORY.ROTATIONAL,
-        gcd: { base: 1500 },
+        gcd: { base: 500, minimum: 1 }, // Tick gcd with Missile Barrage proc
```

Comment:
I think this is a bad idea to do it this way, since it's misleading... You should keep the GCD as the GCD, and then normalize the events so that each tick isn't a cast

Reply:
What do you mean "trigger event"? The first cast event in a series of cast events in a short timespan should be enough, then you'll also know max amounts of ticks, so you also have information necessary to split chains of casts if that's happening in a log.

Reply:
Added normalizer